exporter.go

```diff
@@ -48,21 +49,45 @@ func (e *Exporter) Export(
 	meta.RunImage.Reference = runImageRef
 	meta.Stack = stack
 
-	meta.App.SHA, err = e.addLayer(workingImage, &layer{path: appDir, identifier: "app"}, origMetadata.App.SHA)
-	if err != nil {
-		return errors.Wrap(err, "exporting app layer")
+	var buildMetadata BuildMetadata
+	sliceLayers := map[string]identifiableLayer{}
+
+	metadataTomlPath := filepath.Join(layersDir, "config", "metadata.toml")
+	_, err = toml.DecodeFile(metadataTomlPath, &buildMetadata)
```

Comment:
It looks like we are decoding the metadata file twice (also on line 170)

Reply:
good catch!